### DISCUSSION TITLE
Oqtane 2.0

### DISCUSSION THREAD

**Discussion by dtquocbao** [oqtane/oqtane.framework#880] [Q&A]
Thanks all the team for your effort on Oqtane 2.0 and .NET 5!

I am hunger to try it out by upgrading my current pet project. However I got this issue. Any idea?

![image](https://user-images.githubusercontent.com/4122092/98904884-4a36b080-246f-11eb-905a-37c71c4a5518.png)

**Comment by hishamco** [comment]
I test it in my end, everything is work great, could you please run `dotnet restore` and rebuild the solution

**Reply by dtquocbao** [reply]
I tried but no joy

**Reply by hishamco** [reply]
Strange!! could you please show us `Oqtane.Client.csproj` and what's the result of `dotnet --version`?

**Comment by sbwalker** [comment]
@rockydant did you get the latest code from the master branch - or did you download the 2.0 release?

**Reply by dtquocbao** [reply]
I did download the nupkg and tried to upload using built-in upgrade function as well as run from nuget package in VS.

**Reply by sbwalker** [reply]
Can you try the manual upgrade steps: https://www.oqtane.org/Resources/Blog/PostId/543/upgrading-oqtane

**Reply by dtquocbao** [reply]
So are you saying that I can run it from VS and try with manual upgrade steps?

**Reply by sbwalker** [reply]
Note that you need to install .NET 5 as well 

- Install .NET 5 SDK (v5.0.100) - https://dotnet.microsoft.com/download/dotnet/5.0
- Install the latest edition (v16.8 or higher) of Visual Studio 2019 (Community, Professional, or Enterprise Editions) with the ASP.NET and web development workload enabled. Oqtane works with ALL editions of Visual Studio from Community to Enterprise. If you do not have a SQL Server installation available already and you wish to use LocalDB for development, you must also install the .NET desktop development workload.

**Reply by dtquocbao** [reply]
Yeah, I did all and also try to upgrade the reference packages too

**Reply by sbwalker** [reply]
You have not provided any details about your environment. ie. do you have a local installation with source code? Do you have a local installation without source code ( ie. based on an installable release package )? Do you have an installation published to a server? To Azure? It is not possible to provide upgrade instructions without knowing the environment details.

**Comment by dtquocbao** [comment]
Thanks for your kind support and Sorry for this late response,

I was using VS 2019 with .net 5 sdk (Windows OS environment, local IIS server). I was working on the source code from git hub (v1.0.4) and trying to upgrade it to v 2.0 by downloading .nupkg files. I want to try the new performance of Oqtane as well as see the enhancements such as UI or code refactor, etc... You can see my previous screenshot that I was trying with VS.

**Reply by sbwalker** [reply]
The Oqtane framework source code does not contain references to the Oqtane Client, Server, or Shared Nuget packages - those packages are only intended to be used as references in modules. So if you added your own references to those 2.0 Nuget packages I have no idea what would happen but I assume it would cause problems. You can’t upgrade the 1.0.4 source code in this manner. You need to get the 2.0 source code and overwrite the 1.0.4 source code.
